### PR TITLE
crowbar: Fix blockui appearing nearly all the time (bsc#966356)

### PR DESCRIPTION
**Pull request by MaximilianMeister**
selectors were OR chained, not AND

https://bugzilla.suse.com/show_bug.cgi?id=966356

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

**Comment by rsalevsky** [comment]
+1

File: crowbar_framework/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ jQuery(document).ready(function($) {
   });
 
   setBlockUI('submit', $('[data-blockui]'));
-  setBlockUI('click', $('body[class="installer/upgrades"] [data-blockui]'));
+  setBlockUI('click', $('body[class="installer/upgrades"]').find('[data-blockui]'));
```
Comment (tboerger) [inline crowbar_framework/app/assets/javascripts/application.js:107]: If you use proper selectors it should work. Without testing it I'm pretty sure `body[class="installer/upgrades"] > [data-blockui]` should work.

**Pull request merged**